terraform/prometheus/prometheus.yaml

```diff
@@ -0,0 +1,14 @@
+global:
+  scrape_interval:     15s
+  evaluation_interval: 15s
+
+scrape_configs:
+  - job_name: 'prometheus'
+    static_configs:
+      - targets: [ 'localhost:9090' ]
+  - job_name: 'tempo'
+    static_configs:
+      - targets: [ 'tempo:3200' ]
```

Comment:
Is there a variable that could change this value in the tempo module or the metrics exporter ? If there is we should paramterize the two values.

Reply:
You mean to parametrize the port number?(if it can change)

Reply:
I mean : If we already have an option that allows to change the port, we need to make sure that we use this parameter. If we do not have something like that, it is ok for me if we let it like that